src/components/Editor/Tabs.css

```diff
@@ -7,7 +7,6 @@
   width: 100%;
   height: 30px;
   display: flex;
-  align-items: flex-end;
```

Comment:
This seems out of place?

Reply:
It pushes the toggle buttons to the bottom rather than the middle. I had assumed that was necessary to align the tabs with the bottom but they didn't move when this was removed.